.github/workflows/ci.yml

```diff
@@ -619,3 +619,18 @@ jobs:
           done
 
           echo "::endgroup::"
+          
+  trigger-dependent-workflows:
+    runs-on: ubuntu-latest
+    needs:
+      - services-docker-image-publish
+      - docker-image-publish
+    steps:
+      - name: Send context to seperate pipeline
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.EXTERNAL_RELEASE_TOKEN }}
```

Comment:
I named this rather randomly, as idk how secrets are named or set up. Would appreciate some guidance here 🙏🏼

Reply:
ah forgot to add, need a token with content write permission on [RocketChat/Release.Distributions](https://github.com/RocketChat/Release.Distributions) repo.

Reply:
```suggestion
          token: ${{ secrets.DISTRIBUTION_TOKEN }}
```